spec/integration/entities_spec.rb

```diff
@@ -149,7 +149,7 @@
     fill_in 'name', with: nil
     fill_in 'amount', with: 10.0
     click_button 'ADD TRANSACTION'
-    expect(page).to_not have_content('Name can\'t be blank')
+    expect(page).to have_content('Name can\'t be blank')
```

Comment:
**You have done great job!**

- [ ] However, you have 8 more failing tests to fix. Kindly fix them and make sure if they are passing or not :)

> COMMANDS I used: bundle, rails db:reset, rails assets:precompile, rspec spec
<img width="717" alt="Screenshot 2023-06-02 at 06 14 16" src="https://github.com/biruk-bereka/Budget-app/assets/69011963/dccd0310-dba6-4428-b202-669468f4d728">